src/common/render-xo-item.js

```diff
@@ -182,6 +182,10 @@ const renderXoItem = (item, {
 } = {}) => {
   const { id, type, label } = item
 
+  if (item.removed) {
+    return <span key={id} className='text-danger'> <Icon icon='alarm'size='1' /> {id}</span>
```

Comment:
Missing space between `icon` and `size` prop.

And I don't think it's necessary to add `size='1'`.